src/modules/ie-objects/services/ie-objects.service.ts

```diff
@@ -568,7 +568,8 @@ export class IeObjectsService {
 				...spaces.items.map((space) => space.maintainerId),
 				user.getMaintainerId(),
 			];
-		}
+		} else accessibleVisitorSpaceIds = visitorSpaceAccessInfo.visitorSpaceIds;
```

Comment:
We don't use the short notation for ifs
```suggestion
		} else {
			accessibleVisitorSpaceIds = visitorSpaceAccessInfo.visitorSpaceIds;
		}
```